src/Raven.Client/Documents/Session/ClusterTransactionSession.cs

```diff
@@ -34,6 +35,21 @@ public abstract class ClusterTransactionOperationsBase
 
         private readonly Dictionary<string, CompareExchangeSessionValue> _state = new Dictionary<string, CompareExchangeSessionValue>(StringComparer.OrdinalIgnoreCase);
 
+        private Dictionary<string, long> _missingDocumentsToAtomicGuardIndex;
+        private string _clusterTransactionId;
+
+        internal bool TryGetMissingAtomicGuardFor(string docId, out string changeVector)
+        {
+            if (_missingDocumentsToAtomicGuardIndex?.TryGetValue(docId, out var index) == true)
+            {
+                changeVector = $"TXRN:{index}-{_clusterTransactionId}";
```

Comment:
`ChangeVectorParser.TrxnTag`

Reply:
Cannot use that here, I"m afraid, that is on the _server_ only, not exposed to the client.

Reply:
Done